app/db/index.js

```diff
@@ -8,16 +8,15 @@ const connect = () => {
   const config = {
     connection: () => {
       if (URL) return URL;
-      return `postgres://${username}:${password}@localhost:5432/${name}`;
+      return `postgres://${username}:${password}@localhost:${port}/${name}`;
     },
     options: {
       host,
       dialect,
-      port,
       logging: false,
     },
   };
-  if (server.environment !== 'development') {
+  if (server.environment === 'staging') {
```

Comment:
We'll want this to be active in production as well.

Reply:
Added this to the if statement